packages/@aws-cdk/aws-ssm/lib/parameter.ts

```diff
@@ -145,9 +152,12 @@ abstract class ParameterBase extends Resource implements IParameter {
   }
 }
 
-const STRING_PARAM_TYPE = 'String';
-const SECURE_STRING_PARAM_TYPE = 'SecureString';
-const STRINGLIST_PARAM_TYPE = 'StringList';
+export enum ParameterType {
+  STRING = 'String',
```

Comment:
add documentation to each one (paste from AWS docs)

Reply:
I couldn't find a proper documentation page describing the types, but I pieced something together